client/src/pages/DisplayAnimalsPage/Pagination.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Grid, Typography } from '@mui/material';
+
+const Pagination = (props) => {
+  const { totalPages, onPageNumberClick: handlePageNumberClick } = props;
+  const pageNumbers = [];
+
+  for (let i = 1; i <= totalPages; i++) {
+    pageNumbers.push(i);
+  }
```

Comment:
try nga i set nalang daan ang pageNumbers to ana nga array, sa ES6 daghan na kaayog mga functions nga rare nata mo gamit ug mga "primitive 😂" loops nga ani

`const pageNumbers = [ ...Array(N).keys() ].map( i => i+1);`

Reply:
resolved